db/seeds.rb

```diff
@@ -77,21 +77,21 @@ def get_seed_specification
 
     if seed_environment_value.blank?
       seed = 0
-      puts "\nENV['DB_SEEDS_RANDOM_SEED'] not set to 'random' or a number; setting seed to 0.\n\n"
+      # puts "\nENV['DB_SEEDS_RANDOM_SEED'] not set to 'random' or a number; setting seed to 0.\n\n"
     elsif seed_environment_value.casecmp("random") == 0
       seed = Random.new_seed
-      puts "\n'random' specified in ENV['DB_SEEDS_RANDOM_SEED']; setting seed to randomly generated value #{seed}.\n\n"
+      # puts "\n'random' specified in ENV['DB_SEEDS_RANDOM_SEED']; setting seed to randomly generated value #{seed}.\n\n"
     else
       seed = seed_environment_value.to_i
-      puts "\nUsing random seed #{seed} specified in ENV['DB_SEEDS_RANDOM_SEED'].\n\n"
+      # puts "\nUsing random seed #{seed} specified in ENV['DB_SEEDS_RANDOM_SEED'].\n\n"
     end
     seed
   end
 
   def report_object_counts
-    puts "\nRecords written to the DB:\n\nCount  Class Name\n-----  ----------\n\n"
+    # puts "\nRecords written to the DB:\n\nCount  Class Name\n-----  ----------\n\n"
     active_record_classes.each do |klass|
-      puts "%5d  %s" % [klass.count, klass.name]
+      # puts "%5d  %s" % [klass.count, klass.name]
```

Comment:
The `puts` statements are useful when first setting up the application or when re-seeding. Please revert these.

Reply:
OK, I reinstated the puts commands.